src/Stratis.Bitcoin.Features.WatchOnlyWallet/Controllers/WatchOnlyWalletController.cs

```diff
@@ -34,7 +34,7 @@ public WatchOnlyWalletController(IWatchOnlyWalletManager watchOnlyWalletManager)
         /// <param name="address">The base58 address to add to the watch list.</param>
         [Route("watch")]
         [HttpPost]
-        public IActionResult Watch([FromQuery]string address)
+        public IActionResult Watch([FromBody]string address)
```

Comment:
This needs to be checked with the wallet (Tumblebit in particular) to make sure it doesn't break anything. If it does, then please create an issue reminding them to update the code at the same time they get the latest FN version.

Reply:
Cool I'll do that

Reply:
Jeremy if you didn't remember that then this could potentially have caused a problem in Tumblebit.

Thinking out load - is there any mileage is adding a remark at controller class level if the controller is used in other Stratis related Projects ? This would help if devs make a change that has a dependency.

For Breeze-Tumblebit in particular this problem would go away if and when it starts referencing the FullNode nuget packages.

Reply:
If anything, I'd add a quick test with a name like "TestControllerIsDoingWhatTumblebitExpects" and make sure it breaks when a breaking change is made. Then the person changing it will ask why this test is there and hopefully get some help.
But if tumblebit is using a nuget package, then before upgrading to a new package it should be tested and that issue should come out.
I think creating dependency to downstream projects through comments is not the way forward. It can get messy if you start using the same project in many others (which is what we should hope for), and then have to update your comments in top level project everytime a downstream projects evolve...